extensions/amp-story/1.0/amp-story-localization-service.js

```diff
@@ -42,7 +33,7 @@ export function localize(context, key) {
  * @return {!Promise}
  */
 export function localizeTemplate(template, context) {
-  const localizationService = getLocalizationService(context);
+  const localizationService = Services.localizationForDoc(context);
```

Comment:
Need to use this function so it doesn't bundle the localization service with all the components